developerportal/templates/molecules/item-topic-link.html

```diff
@@ -16,4 +19,18 @@ <h4 class="no-underline">{{ topic.title }}</h4>
       {{ topic.description | richtext }}
     {% endif %}
   </div>
-</a>
+</a> {% endcomment %}
```

Comment:
Is everything between `{% comment %}` and `{% endcomment %}` excluded from what is sent to the browser? Or are these not that type of comment :)

Reply:
Yep, stuff inside `{% comment %}` is excluded, but this has since been amended/reverted/updated